python/py-requests-oauthlib/Portfile

```diff
@@ -30,6 +30,9 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-requests \
                     port:py${python.version}-oauthlib
 
+    depends_test-append \
+                    port:py${python.version}-pip
```

Comment:
why is `pip` needed as a test dependency? Looking at [upstream](https://github.com/requests/requests-oauthlib/blob/b0857c856dc76cc375f72d7c2c5b773bdc55526e/setup.py#L69) it seem that `mock` and `requests-mock` are needed, but `pip` isn't mentioned

Reply:
@reneeotten without `pip` it fails as:
```
error: Command '['/opt/local/Library/Frameworks/Python.framework/Versions/3.10/bin/python3.10', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/opt/local/var/macports/build/_Users_catap_src_macports-ports_python_py-requests-oauthlib/py310-requests-oauthlib/work/.tmp/tmp6qyykcpz', '--quiet', 'mock']' returned non-zero exit status 1.
Command failed:  cd "/opt/local/var/macports/build/_Users_catap_src_macports-ports_python_py-requests-oauthlib/py310-requests-oauthlib/work/requests-oauthlib-1.3.1" && /opt/local/Library/Frameworks/Python.framework/Versions/3.10/bin/python3.10 setup.py --no-user-cfg test 
Exit code: 1
Error: Failed to test py310-requests-oauthlib: command execution failed
```

Reply:
Yes, but if you read that error message it's because it tries to install `mock`, so once you list the correct test dependencies it will not need to use `pip` anymore

Reply:
@reneeotten indeed!

Reply:
great! Please do add both dependencies that are listed by upstream; you'll need `py-mock` as well (see link above)